Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/StatsItem.kt

```diff
@@ -75,6 +76,22 @@ data class IncidenceStats(
     }
 }
 
+data class LocalIncidenceStats(
```

Comment:
I think this class should contain a federal-state id or translated name

Reply:
done :)